src/main/java/exercise_3/FoodItem.java

```diff
@@ -0,0 +1,15 @@
+package exercise_3;
+
+public class FoodItem {
+    String name;
```

Comment:
Please add "private" modifiers for the attributes 
```suggestion
    private String name;
```